zebrad/tests/common/configs/v1.9.0.toml

```diff
@@ -73,6 +73,8 @@ Heartwood = 903_800
 Canopy = 1_028_500
 NU5 = 1_842_420
 NU6 = 2_000_000
+# FIXME: Use a proper value for NU7.
```

Comment:
This value is already okay. These test configs are just meant to ensure that valid past configs can still be parsed whenever the config is changed and, secondarily, to provide examples.

```suggestion
```